packages/sources/tradingeconomics-test/src/endpoint/price.ts

```diff
@@ -0,0 +1,63 @@
+import { makeLogger } from '@chainlink/external-adapter-framework/util'
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { EndpointTypes } from './price-router'
+import { PriceEndpointParams } from '@chainlink/external-adapter-framework/adapter'
+const logger = makeLogger('TradinEconomics HTTP')
+
+const transformInput = (input: PriceEndpointParams) => {
+  const base = input.base.replace(' ', '').split(',')
+  const quote = input.quote.replace(' ', '').split(',')
+  return base
+    .map((element, index) => {
+      return `${element}:${quote[index]}`
+    })
+    .join(',')
+}
+
+export const batchTransport = new HttpTransport<EndpointTypes>({
+  prepareRequests: (params, config) => {
+    return params.map((input) => {
+      const symbol = transformInput(input)
+      const requestConfig = {
+        baseURL: config.API_ENDPOINT,
+        url: `symbol/${symbol}`,
+        params: {
+          c: `${config.API_CLIENT_KEY}:${config.API_CLIENT_SECRET}`,
+        },
+      }
+      return {
+        params: [input],
+        request: requestConfig,
+      }
+    })
+  },
+  parseResponse: (params, res) => {
+    const entry = {
+      params: params[0],
+    }
+    if (!res.data) {
+      const errorMessage = `Tradingeconomics provided no data for token "${params[0].base}"`
```

Comment:
Nitpick, but these likely won't be "tokens"

```suggestion
      const errorMessage = `Tradingeconomics provided no data for "${params[0].base}"`
```